### DISCUSSION TITLE
Apple Encrypted Archive (.aea) support?

### DISCUSSION THREAD

**Discussion by Wrestor** [aonez/Keka#1352] [Q&A] [answered]
I see that this [issue](https://github.com/aonez/Keka/issues/829) was closed, even though `.aea` support(en/decryption) is not implemented yet? Is there anything preventing its implementation?

**Comment by aonez** [comment]
Hi @RevENGE-code,

I was not aware of that extension. What command/app produced it? Or is it mentioned in some docs?

Renaming to `.aar` should work for your, let me know.

**Reply by aonez** [reply, marked as answer]
Just saw Archive Utility (aka BOOM) creates such extensions. I'll add support for it in the next revision.

Meanwhile you can extract those dropping them on the extraction zone in the Keka window (bottom zone), using the contextual menu item `Extract with Keka` or renaming the extension.
